frontend/src/pages/DebugPage/DebugPage.tsx

```diff
@@ -1,9 +1,14 @@
-import React from 'react'
+import React, { useEffect } from 'react'
 import { DebugLogController } from '../../controllers/DebugLogController'
 import { Typography } from '@material-ui/core'
 import { Page } from '../Page'
+import Analytics from '../../helpers/Analytics'
 
 export function DebugPage() {
+  useEffect(() => {
```

Comment:
I think maybe a more elegant solution to page tracking would be to provide a custom useHistory hook... maybe `useAndTrackHistory` where we just track the react router history changes. Then we don't have to have this useEffect on every page load.

Reply:
I like this idea if we can get the page name.  lets talk

Reply:
we decided to do this later as it would be a large change and works as is for now.